### PR TITLE
Rework README

### PR DESCRIPTION
**Pull request by tiffanychu90**
* Reorganize README to include all data sources
* Add hospital, testing data + sources
* Deprecate old shelter layers
* Double check shelter feature layers are public and working

### CODE REVIEW

File: dags/public-health/covid19/README.md [hunk 1]
```diff
@@ -36,30 +95,33 @@ JHU initially published US county-level data until 3/10/2020. On 3/10, JHU start
...
 
-* [Global province-level time-series feature layer](http://lahub.maps.arcgis.com/home/item.html?id=20271474d3c3404d9c79bed0dbd48580)
 
-* [Global province-level current date's feature layer](http://lahub.maps.arcgis.com/home/item.html?id=191df200230642099002039816dc8c59)
+## Shelter Data
 
-* [US county-level time-series feature layer](http://lahub.maps.arcgis.com/home/item.html?id=4e0dc873bd794c14b7bd186b4b5e74a2)
+The DAG `shelter-to-esri.py` takes the Rec & Parks (RAP) shelter census (collected at 8 intervals a day) and pushes it into the City of LA GeoHub by merging it with the the official shelter data from LA Sanitation (LASAN) and RAP GIS staff. The report intervals are: 6:30am, 10:30am, 11:30am, 12:30pm, 1:30pm, 2:30pm, 3:30pm, 4:30pm, and 8:30pm. We do some timezone data cleaning and publish.
```
Comment (ian-r-rose) [inline dags/public-health/covid19/README.md:109]: Are these times right? Aren't we just doing hourly?

Comment (tiffanychu90) [reply dags/public-health/covid19/README.md:109]: @hunterowens Are shelter census forms taken hourly now?

**Comment by ian-r-rose** [comment]
Ugh, sorry @tiffanychu90, I introduced some merge conflicts because #235 and #236 changed esri layer ids.

File: dags/public-health/covid19/README.md (hunk 1 above)
Comment (hunterowens) [reply dags/public-health/covid19/README.md:109]: the ETL runs hourly but to my knowledge RAP hasn't changed when they instruct staff to report the data in. we expose this in the reporting in the "report time:" field.

**Comment by tiffanychu90** [comment]
@ian-r-rose That's fine. I'll address merge conflicts. This PR isn't ready yet, until it includes hospital and testing sections.

**Comment by tiffanychu90** [comment]
@ian-r-rose Rebased on #235 and #236, but I can wait until Brendan's #227 is merged in to rebase again. Already incorporated stuff from what he added to README already.

**Pull request merged**